### ISSUE TITLE
E605: Exception not caught: NERDTree.InvalidArgumentsError

### ISSUE THREAD

**Issue by mpfusion** [scrooloose/nerdtree#265]
### Steps to reproduce

My `.vimrc`:

```
set nocompatible
filetype plugin indent on
let g:NERDTreeQuitOnOpen = 1
let g:NERDTreeChDirMode  = 2
```

Execute:

```
cd
mkdir /tmp/somedir
touch /tmp/somedir/somefile
vim
```

From within vim:

```
:cd /tmp/somedir
:NERDTreeFind
<cr>j<cr> to open somefile
:bd
:NERDTreeFind
```

The following error is thrown:

```
NERDTree: 1 file(s) could not be loaded into the NERD tree
Error detected while processing function nerdtree#findAndRevealPath..149:
line    2:
E605: Exception not caught: NERDTree.InvalidArgumentsError: /tmp should be under /
```

**Comment by PhilRunninger** [comment]
I can't reproduce this. Is this still happening for anyone else?

**Comment by tony** [comment]
I likely either 

- a. worked around it 
- b. stopped having it due to it being fixed 
- c. system config settings changed since

i'm ok with closing unless it flairs up again

**Comment by mkhuda** [comment]
This error sometimes occurs when I save the markdown file `.md`.

**Comment by zoumi** [comment]
1,Create file E:\test.c

2,Put this to vimrc
autocmd! VimEnter * nested call DoWhenVimEnter()

func! DoWhenVimEnter()
    execute 'edit E:/test.c'
    exec ':NERDTreeFind'
    return
endfunc

3,Start gvim

I'm using gvim 8.2 64bit on window7.

**Comment by zoumi** [comment]
![image](https://user-images.githubusercontent.com/5162901/79765560-5b946000-8359-11ea-96d8-bddf9df26254.png)

**Comment by PhilRunninger** [comment]
@zoumi, This should be fixed by PR #1122.
